include/ibamr/INSVCStaggeredHierarchyIntegrator.h

```diff
@@ -570,7 +570,6 @@ class INSVCStaggeredHierarchyIntegrator : public INSHierarchyIntegrator
     SAMRAI::tbox::Pointer<SAMRAI::pdat::CellVariable<NDIM, double> > d_P_var;
     SAMRAI::tbox::Pointer<SAMRAI::pdat::SideVariable<NDIM, double> > d_F_var;
     SAMRAI::tbox::Pointer<SAMRAI::pdat::CellVariable<NDIM, double> > d_F_cc_var;
-    SAMRAI::tbox::Pointer<SAMRAI::pdat::CellVariable<NDIM, double> > d_Q_var;
```

Comment:
I would consider leaving `Q` here --- its role is very similar to `F`.

Reply:
It is there. It is moved a bit down to line 605 with a comment, as to what `Q` represents. I think we were getting confused with `Q` of advection-diffusion integrator, so it separated it out from the block of variables.  

https://github.com/IBAMR/IBAMR/blob/phase_change_patch_1/include/ibamr/INSVCStaggeredHierarchyIntegrator.h#L605-L607

Reply:
Yes, I saw it was moved --- I was just suggesting to keep it with F because it is the "same kind of variable" as F.